### PR TITLE
[WIP] Starting csv parsing logic

### PR DESCRIPTION
**Pull request by softwarewright**

### CODE REVIEW

**Comment by aiwilliams** [comment]
I see the `[WIP]` title and wondered if you are aware of the the Draft PR feature in GitHub? https://github.blog/2019-02-14-introducing-draft-pull-requests/

File: packages/integration-sdk-parser/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# @jupiterone/integration-sdk-parser
```
Comment (aiwilliams) [inline packages/integration-sdk-parser/README.md:1]: FWIW, I thought this project was going to parse CSV files and convert them to entities/relationships. As it's actually converting entities/relationships into a CSV, I wonder if what we should have is a `CSVGraphObjectStore`, somewhere alongside this https://github.com/JupiterOne/sdk/tree/master/packages/integration-sdk-runtime/src/storage ?

Comment (aiwilliams) [reply packages/integration-sdk-parser/README.md:1]: I may have spoken too soon, I'm not entirely sure of the goal (couldn't find an Issue with "csv" in it describing what we're going for, nor anything in https://github.com/JupiterOne/sdk/blob/master/docs/integrations/development.md that describes "csv").

**Comment by softwarewright** [comment]
> I see the `[WIP]` title and wondered if you are aware of the the Draft PR feature in GitHub? https://github.blog/2019-02-14-introducing-draft-pull-requests/

Nope did not realize that was a thing, thanks @aiwilliams. I'll change it to that instead.

File: packages/integration-sdk-parser/README.md (hunk 1 above)
Comment (softwarewright) [reply packages/integration-sdk-parser/README.md:1]: This code is going to completely change, I didn't push what I had yesterday. I'm working on adding import/export of entities and relationships. We are going to do a bulk download of all entities in json and convert them to csv files for the export command. As for import, we will import all entities and relationships from the exported csv files into an account.

**Pull request closed without merging**